src/main/java/com/checkmarx/jenkins/tools/CheckmarxInstaller.java

```diff
@@ -118,6 +125,22 @@ private FilePath installCheckmarxCliAsSingleBinary(FilePath expected, Node node,
         return expected;
     }
 
+    private String getProxy(Node node) {
```

Comment:
have it in a proxy class too?

Reply:
do you want me to exclude it to a new class?
it is just a few lines...